homeassistant/helpers/entity_platform.py

```diff
@@ -80,6 +80,22 @@ def __call__(
         """Define add_entities type."""
 
 
+class AddConfigEntryEntitiesCallback(Protocol):
+    """Protocol type for EntityPlatform.add_entities callback."""
+
+    def __call__(
+        self,
+        new_entities: Iterable[Entity],
+        update_before_add: bool = False,
+        *,
+        subentry_id: str | None = None,
```

Comment:
I think we should rename this parameter across the module.
```suggestion
        config_subentry_id: str | None = None,
```